src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -593,7 +593,6 @@ public void populateFieldsToGenerateInQuestion(FeedbackQuestionAttributes feedba
         case NONE:
             break;
         case STUDENTS:
-            //fallthrough
```

Comment:
Not sure if removing a `//fallthrough` comment in Java is a good idea in the case of fallthrough; it does has its own semantic contribution.